dev-docs/bidders/adprime.md

```diff
@@ -5,11 +5,11 @@ description: Prebid Adprime Bidder Adapter
 biddercode: adprime
```

Comment:
you need to add title: AdPrime or something similar to the front matter. We use this variable for the bid adapter page display and also for the search function.